### DISCUSSION TITLE
vercel 部署失败 问题

### DISCUSSION THREAD

**Discussion by bobtmee** [walinejs/waline#1574] [Q&A] [answered]
vercel 显示下面的错误 ：
Error: No Output Directory named "dist" found after the Build completed. You can configure the Output Directory in your Project Settings.
20:47:54.138 | Learn More: https://vercel.link/missing-public-directory

我是直接 在vercel 设置 复制 https://github.com/walinejs/waline 这个地址 对吗？

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/guide/get-started.html#vercel-%E9%83%A8%E7%BD%B2-%E6%9C%8D%E5%8A%A1%E7%AB%AF

**Reply by bobtmee** [reply]
谢谢 成功解决了
